sources/org.osbuild.ostree

```diff
@@ -107,10 +111,13 @@ class OSTreeSource(sources.SourceService):
         if not gpg:
             remote_add_args = ["--no-gpg-verify"]
 
+        if remote.get("contenturl", ""):
```

Comment:
```suggestion
        if "contenturl" in remote:
```

This is not equivalent to what you have, but I would argue checking for a Truthy-string is actually not what we want to do (?). If it is, I would still suggest to drop the second argument, since `get()` by default returns `None` which evaluates to false.

(Sorry, missed that on the first review!)